_posts/developers/pod-server/2019-01-01-00_overview.md

```diff
@@ -1,114 +1,119 @@
-* sample commands using certbot-auto<br />
-`$ wget https://dl.eff.org/certbot-auto`<br />
-`$ mv certbot-auto /usr/local/bin/certbot-auto`<br />
-`$ chown root /usr/local/bin/certbot-auto`<br />
-`$ chmod 0755 /usr/local/bin/certbot-auto`<br />
-`$ cd /usr/local/bin/`<br />
-`$ ./certbot-auto certonly \`<br />
-`--manual \`<br />
-`--preferred-challenges=dns \`<br />
-`--email mail@host.com \`<br />
-`--server https://acme-v02.api.letsencrypt.org/directory \`<br />
-`--agree-tos \`<br />
-`-d example.org -d *.example.org`<br />
-//now DNS-challenge<br />
-`$ chmod -R 755 /etc/letsencrypt/live/`<br />
-
-### Configure a reverse proxy
-
-The Solid server will run as a local Web server. You can directly expose the port it binds to, but you may also choose to access it through a reverse proxy. We provide some guidance for the following reverse proxies:
-- [Apache](/for-developers/pod-server/apache)
-- [Nginx](/for-developers/pod-server/nginx)
-
-### Install Node Solid Server
-
-- First, if it is not already done on your machine, install Node.js (version 12 or above), and npm ([this guide](https://tecadmin.net/install-latest-nodejs-npm-on-debian/) provides detailed instructions). 
-
-- Then, run the command `$ npm install -g solid-server`. After the init-command you'll be prompted a series of questions to configure the server:
-
-```config
-$ solid init
-* ? Path to the folder you want to serve. Default is (./data) /var/www/your.host.example.org/data
-? SSL port to run on. Default is (8443) 8443
-? Solid server uri (with protocol, hostname and port) https://your.host.example.org
-? Enable WebID authentication Yes
-? Serve Solid on URL path /
-? Path to the config directory (for example: /etc/solid-server) (./config) /var/www/your.host.example.org/config
-? Path to the config file (for example: ./config.json) (./config.json) /var/www/your.host.example.org/config.json
-? Path to the server metadata db directory (for users/apps etc) (./.db) /var/www/your.host.example.org/.db
-? Path to the SSL private key in PEM format /etc/letsencrypt/live/your.host.example.org/privkey.pem
-? Path to the SSL certificate key in PEM format /etc/letsencrypt/live/your.host.example.org/fullchain.pem
-? Enable multi-user mode Yes
-? Do you want to set up an email service (y/N) N
-? A name for your server (not required) your.host.example.org
-? A description of your server (not requred) undefined
-? A logo (not required) undefined
-? Do you want to enforce Terms & Conditions for your service (y/N) N
-? Do you want to disable password strength checking (y/N) N
-? The support email you provide for your users (not required) undefined
-config created on /root/config.json
+# Running your own Solid server
+A Solid server allows you to register for a Pod and a WebID.
+<br>
+You can either [use a publicly available server](/users/get-a-pod)
+or follow the steps on this page to set up your own.
+
+A couple of **open-source servers** exist:
```

Comment:
Since we're listing more than a couple and as the list grows, don't want to keep a count:

```suggestion
The following  lists some available **open-source servers**:
```

or

```suggestion
The following **open-source servers** are available:
```